Godeps/LICENSES

```diff
@@ -20228,6 +20293,43 @@ SOFTWARE.
 ================================================================================
 
 
+================================================================================
+= vendor/gopkg.in/tomb.v1 licensed under: =
+
+tomb - support for clean goroutine termination in Go.
+
+Copyright (c) 2010-2011 - Gustavo Niemeyer <gustavo@niemeyer.net>
```

Comment:
hmm... this license doesn't have a recognized name... @cblecker, does the text match a known/acceptable license?

Reply:
ah, looks to match the bsd 3-clause license, but extra eyes are appreciated

Reply:
looks like the 3 clause BSD:
https://opensource.org/licenses/BSD-3-Clause

> Redistributions in binary form must reproduce the above copyright notice,
this list of conditions and the following disclaimer in the documentation
and/or other materials provided with the distribution.

is not exactly something that certain a parties want to comply with.
i'd ask the license experts.

Reply:
we already have 14 other dependencies with this license, fyi

Reply:
in that case we can assume it's OK.

Reply:
FYI https://github.com/kubernetes/community/blob/b1851114db65091eff064cee2b786826b203c231/contributors/devel/sig-architecture/godep.md#reviewing-and-approving-dependency-changes ->
https://github.com/cncf/foundation/blob/master/whitelist-policy.md#approved-licenses-for-whitelist

3 clause BSD is approved

Reply:
nice list, thanks.